.gitignore

```diff
@@ -55,53 +19,8 @@ jspm_packages/
 # Output of 'npm pack'
 *.tgz
 
-# Yarn Integrity file
-.yarn-integrity
-
-# dotenv environment variables file
-.env
-.env.test
-
-# parcel-bundler cache (https://parceljs.org/)
-.cache
-
-# Gatsby files
-.cache/
-
-# Stores VSCode versions used for testing VSCode extensions
-.vscode-test
-
-# CI script debris
-resolutions.json
-
-### OSX ###
-.DS_Store
```

Comment:
This is often a thing for me, might put this in again at some point, but no blocker for now.